ta4j-core/pom.xml

```diff
@@ -38,5 +38,17 @@
             <version>3.17</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.9.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Adding additional dependency without consultation will not be accepted. Furthermore they are not needed for this PR. Please use existing functions of `TestUtils` or existing `junit.junit` dependency

Reply:
@team172011 Okay, I just thought they are okay alongside this PR as they are commonly used in Java land. I'll change not to use them.